pyclang/idf_extension.py

```diff
@@ -24,6 +32,15 @@ def check_clang_toolchain(subcommand_name, args):
                f'set to "{toolchain}".'), file=sys.stderr)
 
     def call_runner(subcommand_name, ctx, args, **kwargs):
+        if not check_esp_clang():
+            print(
+                'ERROR: Espressif Clang was not found on the system.\n'
+                'For installation instructions, visit the documentation:\n'
+                'https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-clang-tidy.html',
+                file=sys.stderr,
+            )
+            sys.exit(1)
```

Comment:
You don't have to address this but it might be simpler to raise SystemExit. That is the same as specifying STDERR in the print and exiting. No backtrace is printed for SystemExit.